BoxOfficeTests/DecodingManagerTests/DecodingManagerTests.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  DecodingManagerTests.swift
+//  DecodingManagerTests
+//
+//  Created by Dasan & Whales on 2023/07/25.
+//
+
+import XCTest
+@testable import BoxOffice
+
+final class DecodingManagerTests: XCTestCase {
+    private var sut: DecodingManager!
+    
+    override func setUpWithError() throws {
+        sut = DecodingManager()
+    }
+
+    override func tearDownWithError() throws {
+        sut = nil
+    }
+    
+    func test_일치하는_dataAsset이없을때_notFoundAsset에러를_던진다() {
+        //given
+        let fileName = "없는파일이름"
+        
+        //when, then
+        XCTAssertThrowsError(try sut.decodeJSON(fileName: fileName) as BoxOffice) { error in
+            XCTAssertEqual(error as! DataError, DataError.notFoundAsset)
+        }
+    }
+    
+    func test_일치하는_dataAsset가있을때_notFoundAsset에러를_던지지않는다() {
+        //given
+        let fileName = "box_office_sample"
+        
+        //when, then
+        XCTAssertNoThrow(try sut.decodeJSON(fileName: fileName) as BoxOffice)
+    }
+    
+    func test_Model타입과_JSON데이터의_key가달라_decoding이_실패했을때_failedDecoding에러를_던진다() {
+        //given
+        let fileName = "box_office_sample_test"
+        
+        //when, then
+        XCTAssertThrowsError(try sut.decodeJSON(fileName: fileName) as BoxOffice) { error in
+            XCTAssertEqual(error as! DataError, DataError.failedDecoding)
```

Comment:
```suggestion
            XCTAssertEqual(error as? DataError, DataError.failedDecoding)
```

여기서는 강제 언래핑을 하지 않아도 비교할 수 있습니다!
참고차 말씀드려요 😄

Reply:
커밋링크 : https://github.com/yagom-academy/ios-box-office/pull/73/commits/3225abb04dc186f61517ed4af3cb9eaaf6ff1d6c

오 감사합니다!! 알려주신 덕분에 수정하였습니다!!🙏